BUILDING.md

```diff
@@ -1,244 +1,102 @@
-# Building Bottlerocket
 
-If you'd like to build your own image instead of relying on an Amazon-provided image, follow these steps.
-You can skip to the [setup guide for Kubernetes](QUICKSTART-EKS.md) or the [setup guide for Amazon ECS](QUICKSTART-ECS.md) to use an existing image in Amazon EC2.
-(We're still working on other use cases!)
-
-## Build an image
-
-### Dependencies
+If you'd like to build your own kit instead of relying on an Amazon-provided one, follow these steps.
 
+## Dependencies
 #### System Requirements
-
 The build process artifacts and resulting images can consume in excess of 80GB in the local directory.
-
-The build process is also fairly demanding on your CPU, since we build all included software from scratch.
-(The first time.  Package builds are cached, and only changes are built afterward.)
+The build process is also fairly demanding on your CPU, since we build all included software from scratch. (This is only done Tte first time.  Package builds are  cached, and only changes are built afterward.)
```

Comment:
```suggestion
The build process is also fairly demanding on your CPU, since we build all included software from scratch. (This is only done the first time. Package builds are cached, and only changes are built afterward.)
```